src/doc/playback.cpp

```diff
@@ -167,7 +167,7 @@ void Playback::handleEnterFrame(const frame_t frameDelta, const bool firstTime)
     case PlayAll:
     case PlayInLoop: {
       const Tag* tag = this->tag();
-      const frame_t frame = m_frame;
+      frame_t frame = m_frame;
```

Comment:
@Gasparoken this can be const:
```suggestion
      const frame_t frame = m_frame;
```
Just as a side note, clang-tidy recommend the east-const notation, but we use the west-const notation, more info in #4361

Reply:
I see that this was already const originally, so there is no need for change in this line.